invenio_users_resources/services/users/tasks.py

```diff
@@ -54,3 +54,13 @@ def unindex_users(user_ids):
             current_users_service.indexer.bulk_delete(user_ids)
         except search.exceptions.ConflictError as e:
             current_app.logger.warn(f"Could not bulk-unindex users: {e}")
+
+
+@shared_task(ignore_result=True)
+def execute_moderation_actions(user_id, action):
+    """Execute moderation actions."""
+    actions = current_actions_registry.get(action, [])
+    for action_callable in actions:
+        # Actions should be callables
+        if callable(action_callable):
+            action_callable(user_id)
```

Comment:
question: do we want all callables to be "all or nothing"? 

Right now, if one fails then all will fail. However, each one of them might (or not) implement its own `uow`. Thus, if the nth callable failed then all the other callables that ran before could have already written to DB (leaving an inconsistent state). 

We can create a `uow` and pass it down to the callable and let them use it or not.

WDYT?